platform/ui/src/components/Icon/getIcon.jsx

```diff
@@ -1,37 +1,42 @@
 import React from 'react';
 // Icons
+
 import arrowDown from './../../assets/icons/arrow-down.svg';
-import seriesActive from './../../assets/icons/series-active.svg';
-import seriesInactive from './../../assets/icons/series-inactive.svg';
 import calendar from './../../assets/icons/calendar.svg';
 import cancel from './../../assets/icons/cancel.svg';
 import chevronDown from './../../assets/icons/chevron-down.svg';
 import chevronRight from './../../assets/icons/chevron-right.svg';
+import iconNavStudies from './../../assets/icons/icon-nav-studies.svg';
```

Comment:
Similarly, I'm not sure the naming should isolate this to studies

Reply:
Cool, just update the naming as well.